### PR TITLE
Replace request with isomorphic-fetch

### PR DESCRIPTION
**Pull request by vmihailenco**

### CODE REVIEW

**Comment by sithmel** [comment]
You still import "request" in reporter.ts and request.ts.

**Comment by vmihailenco** [comment]
Those are typescript definitions - they are not included in generated JavaScript.

**Comment by sithmel** [comment]
@vmihailenco I didn't realised TS was so clever to not include request in this case (I also double checked and it works fine).
I needed to do a couple of changes though (adding comments in the code)

File: package.json [hunk 1]
```diff
@@ -14,6 +14,9 @@
     "airbrake",
     "notifier"
   ],
+  "dependencies": {
+    "isomorphic-fetch": "^2.2.1"
```
Comment (sithmel) [inline package.json]: I don't see the point of adding the dependency here. Given that it should be up to the consumer having this available.

**Comment by vmihailenco** [comment]
I've released this as v1.5.0-beta so it is easier to try it out. We still can make changes if we need them.

File: package.json (hunk 1 above)
Comment (sithmel) [reply package.json]: hold on with that, it looks like isomorphic-fetch is now required? let me double check

Comment (sithmel) [reply package.json]: You latest commit makes this PR very similar to mine :-)
I am adding any further comment in the issue #465

Comment (vmihailenco) [reply package.json]: My thinking is that most people don't have/want to deal with Webpack configuration so making their life easier is better than making bundle smaller + forcing them to learn how to configure Webpack. And some of them simply don't use Webpack at all...

Comment (sithmel) [reply package.json]: Good thinking. That works for me

**Pull request merged**